### DISCUSSION TITLE
When focusing on the beginning and end of the table component, the cursor looks strange.

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [ueberdosis/tiptap#5702] [General Questions] [answered]
<img width="1272" alt="스크린샷 2024-10-06 173225" src="https://github.com/user-attachments/assets/7e83cdc6-c393-4212-8aaf-5bfd85d82966">

Is it possible to make the cursor appear normally like the picture below?

![image](https://github.com/user-attachments/assets/ef87955d-998f-496d-b904-c7b399ca61a5)

**Comment by kart-c** [comment, marked as answer]
This must be the [Gapcursor extension](https://tiptap.dev/docs/editor/extensions/functionality/gapcursor#page-title). This article explains in brief what it is.

**Reply by ryuhangyeong** [reply]
I installed the extension, but the same problem occurs.

**Reply by kart-c** [reply]
I think there was a misunderstanding. For the normal cursor to appear you will need a `paragraph` there, which is not present. The `gapcursor` is just an indication of the focus for block level elements like `table`, `image`, and `video`.

So if you had an image and your editor does not have the `gapcursor`, then when your cursor moves from above the image to below it (assuming the image is the last node in the document), the cursor will not be visible.

**Reply by bdbch** [reply]
Thanks for helping out @kart-c - yeah this is the `Gapcursor` extension. The reason why you see this is because otherwise the user would not be able to insert content at those positions @ryuhangyeong. This seems unnecessary in normal content, however when you have a table at the end of your document you could **not add content below the table**.
